lfucg-exactions/client/js/components/AccountExisting.js

```diff
@@ -55,7 +55,7 @@ class AccountExisting extends React.Component {
                 };
             })(lots));
 
-        const accounts_list = accounts && accounts.length > 0 ? (
+        const accounts_list = !!accounts && !!accounts.accounts && accounts.accounts.length > 0 ? (
```

Comment:
@KellyInnovation Whats the significance behind the `!!`?

Reply:
It is a way to make sure it isn't false (null, undefined, empty in some way)